math/average.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <assert.h>
+
+//average of an array
+
+double average(int avg[]) {
+    double result = 0;
+    int size=0;
+    
+    while(avg[size+1]!='\0'){
```

Comment:
I think this case you can find the array size without a loop. For example: `sizeof(avg) / sizeof(int)` What do you think?

Reply:
I tried it too but i did not get it to work

Reply:
IIRC, `sizeof(avg) / sizeof(int)` won't work here, an array degrades into a pointer to its first element when passed into a function, and dividing the size of a pointer obviously does nothing

Reply:
yes, you are right.